packages/scss/src/components/inputs/field/_field.scss

```diff
@@ -58,6 +58,7 @@
 			display: inline-flex;
 			margin-left: _component("field.input.inline-margin");
 			margin-right: _component("field.input.inline-margin");
+			margin-bottom: 1.4rem;
```

Comment:
It is not a suitable solution as it implies changing the vertical spacing of stuff around. Maybe a negative margin-top ?